spec.bs

```diff
@@ -232,7 +232,122 @@ to conduct an auction to select an advertisement to display to the user, they ca
 {{Window/navigator}}.{{Navigator/runAdAuction()}} function, providing an auction configuration that
 tells the browser how to conduct the auction and which on-device recorded interests are allowed to
 bid in the auction for the chance to display their advertisement.
-  
+
+<h3 id="script-runners">Script Runners</h3>
+
+<xmp class="idl">
+[SecureContext]
+interface InterestGroupBiddingScriptRunner {
+  [NewObject] Promise<undefined> createScriptRunner(USVString script);
+};
+
+[SecureContext]
+interface InterestGroupScoringScriptRunner {
+  [NewObject] Promise<undefined> createScriptRunner(USVString script);
+};
+
+[SecureContext]
+interface InterestGroupReportingScriptRunner {
+  [NewObject] Promise<undefined> createScriptRunner(USVString script);
+};
+
+[SecureContext]
+interface InterestGroupScriptRunner {
+};
+</xmp>
+
+Note: WebIDL interfaces exposed to `*` are not exposed to interest group script runners.

```

Comment:
This cannot be a note. This needs be normative text outside of a note, and use "must" (per RFC guidelines/meanings). I'd say:

> An additional requirement to the interest group script runner globals defined in this specification is that they must not expose any interfaces from other specifications whose [own exposure set](https://webidl.spec.whatwg.org/#own-exposure-set) is the special value "*". The only interfaces that can be exposed inside of the globals defined in this specification are those that explicitly list the global names provided here."

Reply:
Thanks so much for the suggested paragraph!